upgrade/sql/8.1.0.sql

```diff
@@ -38,11 +36,12 @@ SELECT pa.id_product_attribute, l.id_lang, '', ''
 FROM `PREFIX_product_attribute` pa CROSS JOIN `PREFIX_lang` l;
 
 /* Add default redirect configuration and change all '404' to 'default' */
-INSERT INTO `PREFIX_configuration` (`name`, `value`, `date_add`, `date_upd`) VALUES  
+INSERT INTO `PREFIX_configuration` (`name`, `value`, `date_add`, `date_upd`) VALUES
   ('PS_PRODUCT_REDIRECTION_DEFAULT', '404', NOW(), NOW()),
   ('PS_MAINTENANCE_ALLOW_ADMINS', 1, NOW(), NOW())
 ;
 UPDATE `PREFIX_product` SET `redirect_type` = 'default' WHERE `redirect_type` = '404';
 UPDATE `PREFIX_product_shop` SET `redirect_type` = 'default' WHERE `redirect_type` = '404';
-INSERT IGNORE INTO `PREFIX_hook` (`id_hook`, `name`, `title`, `description`, `position`) VALUES
-  (NULL, 'actionLoggerLogMessage', 'Allows to make extra action while a log is triggered', 'This hook allows to make an extra action while an exception is thrown and the logger logs it', '1');
+
+INSERT INTO `PREFIX_feature_flag` (`name`, `state`, `label_wording`, `label_domain`, `description_wording`, `description_domain`, `stability`) VALUES
```

Comment:
Careful, the feature flag actually already exists from 8.0 to 8.1
It's not that it needs to be added, it needs to be updated:
- name must be changed to `New product page`
- stability must be `stable`

In fresh install the state is also forced to 1 in 8.1, but I don't think we should force it for an upgrade.

And finally the `product_page_v2_multi_shop` feature flag must be removed from the DB

Reply:
And thanks for handling this, I didn't find the time to do it myself 😉

Reply:
oh nice catch 👍

Reply:
done 👍